Test/WebTestCase.php

```diff
@@ -79,22 +80,38 @@ protected function tearDown()
     }
 
     /**
+     * @return ContainerInterface
```

Comment:
By this way I can mock ```getContainer``` method and return a mock of ```Container``` instead of using the real container.